app/controllers/application_controller.rb

```diff
@@ -17,6 +18,10 @@ def current_user_id
     warden.user.first.fetch('id')
   end
 
+  def current_user_name
+    @current_user_name || warden.user.first.values_at('first_name', 'last_name').join(' ')
```

Comment:
Heads up, this is not memoizing, needs to be `||=`
Unless this is the desired outcome.

Reply:
Oops, thank you!